src/riak_kv_ttaaefs_manager.erl

```diff
@@ -617,8 +835,25 @@ remote_sender({fetch_clocks, SegmentIDs}, Client, Mod, ReturnFun, NVal) ->
     fun() ->
         case Mod:aae_fetch_clocks(Client, NVal, SegmentIDs) of
             {ok, {keysclocks, KeysClocks}} ->
-                ReturnFun(lists:map(fun({{B, K}, VC}) -> {B, K, VC} end,
-                            KeysClocks));
+                MapFun =
```

Comment:
you use this MapFun at a lot of places. Would it not be better to write a function
`decode_keyclocks` that takes a list and does exactly this.... less code duplication

Reply:
Updated